README.md

```diff
@@ -67,7 +67,7 @@ You will need the following things properly installed on your computer.
 
 * [Git][git]
 * [Node.js][node]
-  * Supported versions: `v16` and `v18`.
+  * Supported versions: `v20`.
```

Comment:
We allow users to run v18 and v20, as per [engines](https://github.com/hashicorp/boundary-ui/blob/main/package.json#L82) config. [More info about engines here](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines).

Reply:
Bet, thanks for the catch Carlos!

Reply:
Oh right. Thanks @calcaide, I forgot we still support 18.